layouts/main.tsx

```diff
@@ -483,15 +505,15 @@ const Layout = ({ children, title = "Livepeer Explorer" }) => {
                               }}
                             >
                               Treasury{" "}
-                              {(totalActiveTreasuryProposals ?? 0) > 0 && (
+                              {(totalActiveProposals ?? 0) > 0 && (
```

Comment:
Is this a typo/unresolved conflict? I thought the variable was named `totalActiveTreasuryProposals` now

Reply:
@victorges, good catch. I think that one was introduced by git when I rebased my previous branch onto the main branch 🤔. I just fixed it 👍🏻.